packages/components/addon/components/hds/reveal/toggle/button.hbs

```diff
@@ -0,0 +1,14 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Button
```

Comment:
I see that you have `aria-expanded` here, I think you may need to also add `aria-controls` as well. Here, a widget is just "something else is revealed or hidden": 

> A button that opens a widget should have `aria-controls` set to the `id` of the expandable widget and `aria-expanded` set to the current state of the widget.

Reply:
Pushed up changes, thanks